proposals/nnnn-interior_cehicle_data_resumption.md

```diff
@@ -0,0 +1,56 @@
+# Interior Vehicle Data resumption
+
+* Proposal: [SDL-NNNN](nnnn-get-interior-data-resumption.md)
```

Comment:
The file name is currently "nnnn-interior_cehicle_data_resumption.md" so should be updated to match what's included here.